src/coldfront_plugin_api/scim_v2/filters.py

```diff
@@ -0,0 +1,26 @@
+from django_scim.filters import UserFilterQuery, GroupFilterQuery
+from django_scim import constants
+from django_scim.utils import get_user_model
+
+
+class ColdfrontUserFilterQuery(UserFilterQuery):
+    """
+    Defines the SCIM filters for Coldfront users
+
+    This class subclasses from `UserFilterQuery`, the class used by  `django-scim` to translate a SCIM filter query (i.e filter=uesrName eq foo) into a raw SQL query.
```

Comment:
Please wrap these lines. Most developers have `word wrap` turned off in their IDE/code editor and viewing this docstring would require scrolling sideways for them. Black uses a maximum of 88 characters per line, that is fine and we can stick with that.

Also make sure that you've removed the unused imports from the files that you've changed.

Reply:
I've wrapped the docstring and removed the unused imports.